abad.go

```diff
@@ -97,4 +121,103 @@ func (a *Abad) evalUnaryExpr(expr *ast.UnaryExpr) (Obj, error) {
 	}
 
 	return num, nil
+}
+
+func (a *Abad) evalExpr(n ast.Node) (types.Value, error) {
+	if !ast.IsExpr(n) {
```

Comment:
Since this method is internal to the package if you already checked before calling this I'm not sure if you need to check it again